cmd/minishift/cmd/config/config.go

```diff
@@ -95,6 +95,7 @@ var (
 	// Host Folders
 	HostFoldersMountPath = createConfigSetting("hostfolders-mountpath", SetString, nil, nil, true, nil)
 	HostFoldersAutoMount = createConfigSetting("hostfolders-automount", SetBool, nil, nil, true, nil)
+	HostFoldersSftpPort  = createConfigSetting("hostfolders-sftp-port", SetInt, []setFn{validations.IsPositive}, nil, true, nil)
```

Comment:
Improvement cange to make sureport is between 1-65536, or 1024-65536

Reply:
> Maybe something we can do later if @hferentschik doesn't have time to spend here.

Right, this would be one of the follow up issues I will create. I don't think it makes sense for me to spend time on these type of things now.

Reply:
> Also we need a config option as hostfolders-type for sshfs or CIFS cc 

For what would you need that? I actually don't think this is needed. I am not sure what you try to achieve with it.

Reply:
I think they are looking for a user-set default. So if I ONLY use sshfs, I can set this as a the 'default' type.

Reply:
I don't think that makes a lot of sense. This is not like the start command where typing flags become tedious. If you go down this road where do you draw the line?